style.css

```diff
@@ -83,6 +83,10 @@
   color: #222222;
 }
 
+h1,h2,h3,h4,p,blockquote {

```

Comment:
Might be better to do the opposite, declare Arial as the global font, then MS Sans Serif for the controls.

Reply:
I am +1 to this! One idea being that wrapping prose in a div or span shouldn’t pixelate it. Wdyt?